### PR TITLE
<BadgeSelectItem /> - refactor storybook page

### PR DESCRIPTION
**Pull request by bamai**
### 🔦 Summary

https://jira.wixpress.com/browse/DSM-878

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6321

File: src/BadgeSelectItem/docs/examples.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+<Layout cols={1}>
+<BadgeSelectItem text='default - general skin' skin='general' />
+<BadgeSelectItem text='standard skin' skin='standard' />
+<BadgeSelectItem text='success skin' skin='success' />
+<BadgeSelectItem text='warning skin' skin='warning' />
+</Layout>
+`;
+
+export const ellipsis = `
+<Box width="300px">
```
Comment (TalZa1810) [inline src/BadgeSelectItem/docs/examples.js:19]: For the `ellipsis` example there's no need to wrap this with `DropdownLayout`. Just display these within `Layout` component (like the previous examples)

File: src/BadgeSelectItem/docs/examples.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+export const defaultExample = `
+<BadgeSelectItem text='Badge Select Item' />
+`;
+
+export const subtitleExample = `
+<BadgeSelectItem text='Badge Select Item' subtitle='subtitle' />
+`;
+
+export const skins = `
```
Comment (TalZa1810) [inline src/BadgeSelectItem/docs/examples.js:9]: Do we want to display all types of skins here?
Let's consult with UX regarding this.

File: src/BadgeSelectItem/docs/index.story.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+export default {
+  category: storySettings.category,
+  storyName: storySettings.storyName,
+  component: BadgeSelectItem,
+  componentPath: '..',
+
+  componentProps: {
+    text: 'Badge select item title',
+    subtitle: 'Badge select item subtitle',
+    suffix: 'Badge select item suffix',
```
Comment (TalZa1810) [inline src/BadgeSelectItem/docs/index.story.js:34]: currently the `suffix` prop is not exposed so no need for this. Let's add the `skin` prop here too.

**Review by TalZa1810 — CHANGES_REQUESTED** [review]
Hey Mai,
I've added few comments. Also, please ask for a UX review regarding the descriptions of the different sections.

Thanks.

**Comment by TalZa1810** [comment]
Hi @milkyfruit ,
This is the `BadgeSelectItem` story page @bamai was working on. I've added some comments regarding this story page. We would like your feedback regarding the descriptions and the examples.

Thanks!

**Review by milkyfruit — CHANGES_REQUESTED** [review]
@TalZa1810 @bamai I reordered the examples and aligned them to be consistent with `ListItemSelect` examples. Please update:

- [x] **Skin**
The component supports 13 different skins. Each skin represents a different status like warning or success. They are reflected in item's prefix.
```
<Layout cols={1}>
  <BadgeSelectItem text="General skin" skin="general" />
  <BadgeSelectItem text="Standard skin" skin="standard" />
  <BadgeSelectItem text="Success skin" skin="success" />
</Layout>;
```

- [x] **Subtitle**
Additional information, like user email or address can be inserted to subtitle area.
```
<BadgeSelectItem text="Badge Select Item" subtitle="subtitle" />;
```

- [x] **Text cropping**
By default component wraps the text. If needed it can be configured to show ellipsis and display full value on hover.
```
<Layout cols={1}>
  <BadgeSelectItem
    text="This is a very very very very long text that is perfect to demonstrate how it will wrap at some point"
    subtitle="This is a very very very very long subtitle that is perfect to demonstrate how it will wrap into multiple lines  at some point"
  />
  <BadgeSelectItem
    ellipsis
    text="This is a very very very very long text that is perfect to demonstrate how it will cropped by ellipsis at some point"
    subtitle="This is a very very very very long text that is perfect to demonstrate how it will cropped by ellipsis at some point"
  />
</Layout>;
```

- [x] **Advanced Example**
All properties work together and can be combined in various ways. It can be rendered as standalone or as part of dropdown.
```
<DropdownLayout
  visible
  inContainer
  selectedId={2}
  options={[
    badgeSelectItemBuilder({
      id: 4,
      text: 'Not Paid',
      subtitle: 'Waiting for a payment',
      skin: 'danger',
    }),
    badgeSelectItemBuilder({
      id: 0,
      text: 'Paid in Person',
      subtitle: 'Cash',
    }),
    badgeSelectItemBuilder({
      id: 2,
      text: 'Paid Plan: Gold',
      subtitle: '8/10 sessions left • Valid until Jan 24 2021',
    }),
    badgeSelectItemBuilder({
      id: 1,
      text: 'Paid Plan: Silver',
      subtitle: '10/10 sessions left • Valid until Dec 22 2020',
    })
  ]}
/>;
```

**Review by TalZa1810 — CHANGES_REQUESTED** [review]
Few other tiny comments.

**Review by TalZa1810 — APPROVED** [review]
Just a tiny note and can be merged:
within the `DropdownLayout`, let's have one option that is disabled.
After this fix, can be merged.

**Pull request merged**